src/transform/commonjs/importCommonjs.js

```diff
@@ -83,7 +83,7 @@ function createImportSpecifier({local, imported}) {
   return new ImportSpecifier({local, imported});
 }
 
-function isVarWithRequireCalls(node) {
+export function isVarWithRequireCalls(node) {
```

Comment:
This should be extracted to a separate file.

Probably should move it out of `commonjs/` dir to distinguish that the code is shared between multiple transforms. So far none of the transforms have referenced functionality of other transforms which keeps things nicely isolated.